src/bin/tikv-ctl.rs

```diff
@@ -116,12 +131,149 @@ fn main() {
             }
         }
         dump_range(db, from, to, limit, cf_name);
+    } else if let Some(matches) = matches.subcommand_matches("mvcc") {
```

Comment:
It's more common to dump a list of different version of the same key and dump the specific version a key.